load-tests/readme.md

```diff
@@ -0,0 +1,41 @@
+# Load Tests
+
+## Setup
+
+These tests use k6 to run. <https://k6.io/docs/>
+
+To run the tests from your local command line install
+k6 in the manner of your choosing
+<https://k6.io/docs/getting-started/installation>
+
+## Running
+
+These tests will use the TEST_SERVICE_CLIENT client for the DEV realm.
```

Comment:
May be good to wrap TEST_SERVICE_CLIENT in backticks to make it highlight as monospaced text.